kind/README.adoc

```diff
@@ -111,41 +126,97 @@ Choices:
 
 == Install
 
-Creating a kubernetes cluster is as simple as executing the following command.
+This script allows creating both a kind cluster with associated ingress controller and a container registry in separate commands.
```

Comment:
Why do you say ".... in separate commands" ?
If this PR installs now using separate parameters the kind cluster AND/OR registry, why, when I execute this command (using bash PR script) -> "curl -s https://raw.githubusercontent.com/snowdrop/k8s-infra/7b7186a8fd90d38195520dc322d1e708422f1f4e/kind/registry.sh | bash -s" nothing is displayed about the commands to be used to install the container registry ?

Reply:
I see what you mean, the help file still refers to the kind cluster. I'll fix that.

```
curl -s https://raw.githubusercontent.com/snowdrop/k8s-infra/7b7186a8fd90d38195520dc322d1e708422f1f4e/kind/registry.sh | bash -s
❌ A command must be provided!!!

Usage: 
	./registry.sh command [parameters,...]

Available commands: 
	install					Install the kind cluster
	remove					Remove the kind cluster

Parameters: 
	-h, --help				This help message

	--provider <provider>			Container Runtime [docker,podman]. Default: docker
	--registry-image-version <version>	Version of the registry container to be used. Default: 2.6.2
	--registry-name <name>			Name of the registry. Default: <cluster_name>-registry
	--registry-password <password>		Registry user password. Default: snowdrop
	--registry-port <port>			Port of the registry. Default: 5000
	--registry-user <user>			Registry user. Default: admin
	--secure-registry			Secure the docker registry. Default: No
	-v, --verbosity <value>			Logging verbosity (0..9). Default: 1
						A verbosity setting of 0 logs only critical events.
```

Reply:
Fixed.

```
./kind/registry.sh                                                                                                                       several-fixes 
❌ A command must be provided!!!

Usage: 
	./registry.sh command [parameters,...]

Available commands: 
	install					Install the container registry
	remove					Remove the container registry

Parameters: 
	-h, --help				This help message

	--provider <provider>			Container Runtime [docker,podman]. Default: docker
	--registry-image-version <version>	Version of the registry container to be used. Default: 2.6.2
	--registry-name <name>			Name of the registry. Default: kind-registry
	--registry-password <password>		Registry user password. Default: snowdrop
	--registry-port <port>			Port of the registry. Default: 5000
	--registry-user <user>			Registry user. Default: admin
	--secure-registry			Secure the docker registry. Default: No
	-v, --verbosity <value>			Logging verbosity (0..9). Default: 1
						A verbosity setting of 0 logs only critical events.

```

@cmoulliard